src/app/ui-components/sidebar-right/sidebar-right.component.ts

```diff
@@ -60,5 +69,42 @@ export class SideBarRightComponent implements OnInit, OnDestroy {
         this.selectedObservable?.subscribe((uuid?: string) => {
             this.router.navigate([uuid ?? "."], {relativeTo: this.route});
         });
+        this.voiceAssistantService.voiceAssistantStateObservable.subscribe(
+            (state: VoiceAssistantState) => {
+                this.voiceAssistantActivationToggle.setValue(state.turnedOn);
+            },
+        );
+    }
+
+    removeCssClass(uuid: string) {
+        const videoSettingsButton = document.getElementById(
+            "dropdownbutton-" + uuid,
+        );
+        videoSettingsButton?.classList.remove("showPopover");
+    }
+
+    addCssClass(uuid: string) {
+        const videoSettingsButton = document.getElementById(
+            "dropdownbutton-" + uuid,
+        );
+        videoSettingsButton?.classList.add("showPopover");
+    }
```

Comment:
can these be removed / replaced with a css rule, that uses a ':hover' selector?

Reply:
resolved by adding :hover selector